### ISSUE TITLE
NSInvalidArgumentException when integrating pod with "use_frameworks!"

### ISSUE THREAD

**Issue by alexandreblin** [mobileprint/ios-print-sdk#1]
I am trying to integrate the Mobile Print SDK into my app but I can't get it to work.

When I try to display the print view controller, the app crashes on [this line](https://github.com/mobileprint/ios-print-sdk/blob/master/Pod/Classes/Private/NSBundle%2BMPLocalizable.m#L23)

This only happens when using `use_frameworks!` in the Podfile (which is required when working with Swift)

It seems to be caused by the fact that when using frameworks, the pod's assets are bundled in the framework and thus not accessible via `[NSBundle mainBundle]`

A possible fix would be to use [resource_bundles](https://guides.cocoapods.org/syntax/podspec.html#resource_bundles) in the podspec instead of [resources](https://guides.cocoapods.org/syntax/podspec.html#resources) which is no longer recommended by CocoaPods

**Comment by alexandreblin** [comment]
Podfile for reference:

```
use_frameworks!

target 'MyTarget' do
    pod 'MobilePrintSDK', '3.0.3'
end
```

This was tested on an empty project with CocoaPods 1.0.0.beta.6 to make sure this was not due to a problem in our Xcode project. The problem also occurs with CocoaPods 0.39.0.

Remove the "use_frameworks!" line, clean the project, delete the app and reinstall to see that it works without it.

**Comment by jbtrask** [comment]
thanks @alexandreblin. we've got a fix for this and we'll be pushing the fix in the next day or two. we solved it by referencing the bundle using the class:

``` objc
NSURL *url = [[NSBundle bundleForClass:[MP class]] URLForResource:HP_PHOTO_PRINT_LOCALIZATION_BUNDLE_NAME withExtension:@"bundle"];
```

we'll look into using `resource_bundles` as that sounds like the more recommended approach.

thanks!

**Comment by jbtrask** [comment]
okay @alexandreblin, we've just released version 3.0.4 of the **MobilePrintSDK** pod. this should work with Swift projects. let me know if it works for you and i'll close this issue. thanks!

**Comment by alexandreblin** [comment]
Perfect! It works fine now. Thanks :+1:
